conda/third_party/dali_opencv/recipe/meta.yaml

```diff
@@ -36,6 +36,8 @@
     host:
       - libjpeg-turbo >=2.0.3
       - libtiff >=4.1.0
+      - libopenjp2 >=2.3.1
```

Comment:
We link OpenCV statically to DALI so the libopenjp2 won't  be propagated. Need to do the same in `conda/recipe/meta.yaml`. Sorry I forgot about this earlier.

Reply:
done